backend/src/plugins/config.ts

```diff
@@ -45,6 +46,7 @@ const configPlugin: FastifyPluginAsync = async (server) => {
     API_PORT: process.env.API_PORT ?? '',
     API_HOST: process.env.API_HOST ?? '',
     SUPPORTED_NETWORKS: process.env.SUPPORTED_NETWORKS ?? '',
+    ADMIN_WALLET_ADDRESS: process.env.ADMIN_WALLET_ADDRESS ?? '0x80a1874E1046B1cc5deFdf4D3153838B72fF94Ac',
```

Comment:
Would it be beneficial here for the 0x address in this short circuit to be a configurable value?